tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -329,6 +329,69 @@ def test_derive_code_mappings_starts_with_app_dot_dot_slash(self):
             assert code_mapping.repository.name == repo_name
 
 
+class TestPhpDeriveCodeMappings(BaseDeriveCodeMappings):
+    def setUp(self):
+        super().setUp()
+        self.platform = "php"
+        self.event_data = self.generate_data(
+            [
+                {"in_app": True, "filename": "/sentry/capybara.php"},
+                {"in_app": True, "filename": "/sentry/potato/kangaroo.php"},
+                {
+                    "in_app": False,
+                    "filename": "/sentry/potato/vendor/sentry/sentry/src/functions.php",
+                },
+            ],
+            self.platform,
+        )
+
+    @responses.activate
+    @with_feature({"organizations:derive-code-mappings-php": False})
+    def test_missing_feature_flag(self):
+        repo_name = "php/place"
+        with patch(
+            "sentry.integrations.github.client.GitHubClientMixin.get_trees_for_org"
+        ) as mock_get_trees_for_org:
+            mock_get_trees_for_org.return_value = {
+                repo_name: RepoTree(Repo(repo_name, "master"), ["sentry/potato/kangaroo.php"])
+            }
+            derive_code_mappings(self.project.id, self.event_data)
+            # Check to make sure no code mappings were generated
+            assert len(RepositoryProjectPathConfig.objects.all()) == 0
```

Comment:
Is this the best way to assert that no code mappings were generated as a result of the feature flag being absent? It feels kind of jank but I'm not sure if there is a better way to do this

Reply:
`assert not RepositoryProjectPathConfig.objects.exists()` is probably what you're looking for: https://docs.djangoproject.com/en/5.0/ref/models/querysets/#django.db.models.query.QuerySet.exists